srcpkgs/olm-python3/template

```diff
@@ -13,8 +13,8 @@ short_desc="Implementation of the Double Ratchet cryptographic ratchet - Python"
 maintainer="Adam Beckmeyer <adam_gpg@thebeckmeyers.xyz>"
 license="Apache-2.0"
 homepage="https://gitlab.matrix.org/matrix-org/olm"
-distfiles="https://gitlab.matrix.org/matrix-org/olm/-/archive/${version}/olm-${version}.tar.bz2"
-checksum=a0482f8c8e80eb11578e8a8655b677fe943b86ac33affd46d67dff4261afeb6a
+distfiles="https://gitlab.matrix.org/matrix-org/olm/-/archive/${version}/olm-${version}.tar.gz"
```

Comment:
Why bigger archives?

Reply:
I thought we prefer `tar.gz`.
I will change it back